templates/books-show.html

```diff
@@ -0,0 +1,20 @@
+<div ng-model="book in booksShowCtrl" class="container">
+  <h3>{{booksShowCtrl.book.title}}</h3>
+  <div class="row">
+    <div class="col-sm-6">
+      <img ng-src="{{booksShowCtrl.book.image}}" class="image-responsive">
+    </div>
+    <div class="col-sm-6">
+      <label ng-click="editing = !editing"><h4 class="bold">Author:</h4><h4 ng-class="editing-label" class="underline">{{booksShowCtrl.book.author}}</h4></label>
+      <input ng-show="editing">
+      <label ng-click="editing = !editing"><h4 class="bold">Release Date:</h4><h4 ng-class="editing-label" class="underline">{{booksShowCtrl.book.releaseDate}}</h4></label>
+      <input ng-show="editing">
+      <p>
+        Click on an item to edit.
+      </p>
+      <div class="buttons">
+        <a href="" ng-click="">Save changes</a>
```

Comment:
If you don't have an href or an ng-click, you don't need those attributes. I wish you'd been able to get this working!